tests/features/collection/collection_overview.feature

```diff
@@ -75,3 +75,6 @@ Feature: Collections Overview
     Then I should see the link "Collections"
     When I click "Collections"
     Then I should see the link "Colonies in space"
+
+    # Debug step.
+    Then I delete the "Colonies in space" collection
```

Comment:
This is not exactly for debugging, but for cleaning up the collection that was made manually. This is a normal part of testing. I'll update the documentation.